src/strongarm.py

```diff
@@ -151,22 +151,21 @@ async def handle_message(websocket, data):
     message_type = data.get("type")
     message_data = data.get("data", {})
 
-    match message_type:
```

Comment:
`match ... case` only available on python >= 3.10 ` should downgrade local anaconda python version so these errors show in dev